README.md

```diff
@@ -30,7 +30,7 @@ regarding these options is available on
 [Intel's website](http://software.intel.com/sites/products/documentation/hpc/mkl/vml/vmldata.htm).
 
```

Comment:
# Example usage
```
julia> using VML
julia> a = rand(10000);
julia>@time  sin.(a);                                                                                                    
 0.159878 seconds (583.25 k allocations: 30.720 MiB, 2.78% gc time)
julia> @time sin(a);                                                                                                      
0.000465 seconds (6 allocations: 781.484 KiB) 
```

Reply:
Note that this example will change after  https://github.com/Crown421/VML.jl/pull/1 , to need an explicit `VML.sin(a)`.

Reply:
Changed